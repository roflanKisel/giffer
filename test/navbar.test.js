import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '../components/navbar';

describe('Navbar', () => {
  it('should renders correctly', () => {
    const wrapper = shallow(<Navbar />);

    expect(wrapper).toMatchSnapshot();
  });
});