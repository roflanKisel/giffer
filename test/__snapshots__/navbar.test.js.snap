// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should renders correctly 1`] = `
<navbar__StyledNavbar>
  <navbar__StyledTitle>
    Giffer
  </navbar__StyledTitle>
  <navbar__StyledNavigation>
    <onlyUpdateForKeys(NavbarItem)
      link="/"
      name="Trending"
    />
    <onlyUpdateForKeys(NavbarItem)
      link="/about"
      name="About"
    />
  </navbar__StyledNavigation>
</navbar__StyledNavbar>
`;
